collection-loader.js

```diff
@@ -1,42 +1,80 @@
-const get = require("lodash/get");
-const flatMap = require("lodash/flatMap");
-const {DEFAULT_STORY_FIELDS} = require("./constants");
+const get = require('lodash/get');
+const flatMap = require('lodash/flatMap');
 
-function loadCollectionItems(client, collections, {storyFields, storyLimits}) {
-  const bulkRequestBody = collections.reduce(
-    (acc, collection) => Object.assign(acc, {
+function loadCollectionItems(
+  client,
+  collections,
+  {storyFields, storyLimits, defaultNestedLimit}
+) {
+  const bulkRequestBody = collections.reduce((acc, collection) => {
+    let limit = storyLimits[get(collection, ['associated-metadata', 'layout'])];
```

Comment:
what will be the default value of the limit ?

Reply:
if the limit is undefined then the limit will be 40